src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -23,11 +23,11 @@
     <div class="input-group">
         {% set append = money_pattern starts with '{{' %}
         {% if not append %}
-            <span class="input-group-addon">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
+            <span class="input-group-addon">{{ money_pattern|convert_encoding(_charset, 'UTF-8')|replace({ '{{ widget }}':''})|raw }}</span>
```

Comment:
I really think this should be `|htmlentities`
we should add the filter to FormExtension

Reply:
Agree. Will patch soonish.

Also templates are UTF8 encoded by default right? Technically we shouldnt have to do any encoding conversion at all here. Instead the rendered template as a whole should be converted depending on the kernel charset i guess.

Implicitly meaning the linked issue is (technically) not supported? Anyway.. htmlentities will settle :)

Reply:
> templates are UTF8 encoded by default right [...] template as a whole should be converted

That's not how Twig deals with charsets - so templates are ASCII that's the only thing we can tell from here.